src/main/java/org/spongepowered/common/SpongeImplHooks.java

```diff
@@ -195,4 +203,28 @@ public static void addItemStackToListForSpawning(Collection<ItemDropData> itemSt
     public static MapStorage getWorldMapStorage(World world) {
         return world.getMapStorage();
     }
+
+    // Crafting
+
+    @Nonnull
```

Comment:
You have to tell your IDE what the NonnullByDefault annotation is, look it up for your ide.